x-pack/plugins/canvas/init.js

```diff
@@ -42,6 +42,7 @@ export default async function(server /*options*/) {
     navLinkId: 'canvas',
     privileges: {
       all: {
+        catalogue: ['canvas'],
```

Comment:
`catalogue` instead of `catalog`, eh? How very Euro of you!

Reply:
I had to be consistent with our [OG implementation](https://github.com/elastic/kibana/blob/master/src/ui/public/registry/feature_catalogue.js) 😄